cvat-ui/src/components/header/header.tsx

```diff
@@ -62,6 +67,34 @@ function HeaderContainer(props: Props): JSX.Element {
                 >
                     Tasks
                 </Button>
+                <Popover placement="bottom" title={content} content={text} trigger="click" overlayStyle={{width : "50vw"}}>
+                <Button
+                    className='cvat-header-button'
+                    type='link'
+                    value='tasks'
+                    onClick={
+                        (): void => {
+                        const t = localStorage.getItem('token');
+                        const token ='Token '+ t.replace(/['"]+/g, '')
+                        const data = axios.get(`${serverHost}/api/v1/server/about`,{ headers: {Authorization : token}}).then(
+                            function(response) {
+                                return response.data;
+                                
+                            }
+                        )
+                        data.then(function(val) { 
+                            const version = 'cvat-version : ' + val.version;
+                            ReactDOM.render(version,document.getElementById('version'))
+                            ReactDOM.render(val.description,document.getElementById('about'))
+                            
+                        });
+                        
+                    }
+                    }
+                >
+                    About
```

Comment:
I believe the best place to put this button is menu in header (near with logout button).